### PR TITLE
Moar Settings

### LINKED ISSUE #44: More Settings
Add more options in Settings page.

Including but not limited to:
1. Option to open the story directly in system browser or integrated browser
2. Night theme
3. Auto expand comments
4. Default home page whether it's Top, Recent, Ask, etc

### PR DESCRIPTION
**Pull request by wiseodd**
Closes #44 

@timotiusnc check out the new options in Settings page.

### CODE REVIEW

File: app/src/main/java/io/pumpkinz/pumpkinreader/NewsListFragment.java [hunk 1]
```diff
@@ -89,11 +93,28 @@ public void forceUnsubscribe() {
...
-        Intent intent = new Intent(getActivity(), NewsDetailActivity.class);
-        intent.putExtra(Constants.NEWS, Parcels.wrap(news));
-
-        startActivity(intent);
+    public void goToNewsDetail(final News news) {
+        SharedPreferences pref = PreferenceManager.getDefaultSharedPreferences(getActivity());
+
+        if (pref.getBoolean(Constants.CONFIG_EXTERNAL_BROWSER, false)) {
+            Intent intent = new Intent(getActivity(), NewsCommentsActivity.class);
+            intent.putExtra(Constants.NEWS, Parcels.wrap(news));
```
Comment (timotiusnc) [inline app/src/main/java/io/pumpkinz/pumpkinreader/NewsListFragment.java]: Can't we extract these duplicate `intent.putExtra` and `startActivity(intent)` to outside the if-else? (line 101,102 and 115,116)

**Comment by timotiusnc** [comment]
What about the Night Theme and Default home page? Is that for next release?

**Comment by wiseodd** [comment]
See #44

**Comment by wiseodd** [comment]
Updated

**Comment by timotiusnc** [comment]
1.1 is ready to gooo!

**Pull request merged**